client/rpcserver/handlers.go

```diff
@@ -1584,4 +1654,88 @@ needed to complete a swap.`,
 	stopMarketMakingRoute: {
 		cmdSummary: `Stop market making.`,
 	},
+	stakeStatusRoute: {
+		cmdSummary: `Get stake status. `,
+		returns: `Returns:
+  obj: The staking status.
+    {
+      ticketprice (uint64): The current ticket price in atoms.
```

Comment:
`ticketprice -> ticketPrice`

Reply:
Ah, fixing these. Was it right to have all of the json tags camel case?

Reply:
Camel-case please.

Reply:
Leaving as is.